### PR TITLE
feat: Add aliases for git diff staged and cached

### LINKED ISSUE #174: Feature Request: Add aliases for git diff commands
## Check list

- [X] I have read through the [README](https://github.com/wfxr/forgit/blob/master/README.md)
- [X] I have the latest version of forgit
- [X] I have searched through the existing issues

## Environment info

- OS
    - [ ] Linux
    - [X] Mac OS X
    - [ ] Windows
    - [ ] Others:
- Shell
    - [ ] bash
    - [X] zsh
    - [ ] fish

## Problem / Steps to reproduce

Add gds alias for `git diff --staged`.
An alias for `git diff --cached` would be great also.

I'd happily open a PR for this.

Thanks for the great plugin!

### PR DESCRIPTION
**Pull request by hermitmaster**
## Check list

- [X] I have performed a self-review of my code
- [X] I have commented my code in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

## Description

Added aliases for `git diff --staged` and `git diff --cached`
## Type of change

- [ ] Bug fix
- [X] New feature
- [ ] Refactor
- [ ] Breaking change
- [x] Documentation change

## Test environment

- Shell
    - [ ] bash
    - [X] zsh
    - [X] fish
- OS
    - [ ] Linux
    - [X] Mac OS X
    - [ ] Windows
    - [ ] Others:

Closes #174

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ You can change the default aliases by defining these variables below.
 ``` bash
 forgit_log=glo
 forgit_diff=gd
+forgit_diff_cached=gdca
```
Comment (cjappl) [inline README.md:134]: I think I would like gdc to mirror gds

**Review by cjappl — CHANGES_REQUESTED** [review]

**Comment by cjappl** [comment]
LGTM besides the small comment. Leaving it up to @wfxr if we want to take the change or not.

Thanks for the contribution :)

File: README.md (hunk 1 above)
Comment (hermitmaster) [reply README.md:134]: I chose gdca to align with the git plugin in omz. I can change to gdc if preferred.

**Comment by wfxr** [comment]
Well, why not just add two lines in your shell's config? I think it's a more elegant way.
```zsh
alias gdca='gd --cached'
alias gds='gd --staged'
```
And more aliases like `alias gdh='gd HEAD'` etc.

**Pull request closed without merging**